.travis.yml

```diff
@@ -46,6 +46,19 @@ jobs:
         # Versions <=9.6.1 is missing function `convert_tuples_by_name_map` needed by our code
         - PG_MAJOR=9.6 PG_MINOR_COMPILE=2 bash -x ./scripts/docker-run-abi-test.sh
 
+    - if: type = cron
+      stage: test
+      install: travis_wait 30 mvn install
+      env:
+        - PG_VERSION=10.0 PG_GIT_TAG=REL_10_0
+        - secure: "uGX2k6j4U6rIIw/+/8Nj2CfVMN6nc0mMhvFK2w1ADRm93Cr3rYo29vfCJYpDGn4W/UIpdl3r15KDJuC/8xF6waHTc9qFpXmUi/VeDLsnsEvBjf3cbK/rKfuzAYZsrRldbwCm+jHUN/aV8q43eGTcrffCFfuhe2gr+sOACCqs/ZxrZlvH2vfPj7pH+FY2xY/J6NvMvmZJljc8rbDJHvuBOu/IIEnhzXNRgOehc7INcaSZM34rx6ktlArJhFWuKnWLfpwXg6v/7So01kGSC0tB9gmL23eXCDrNbIqFrsFn130EfiVUQVClA+j6PGc1N7w+0QWm4h7TlGwwNgthBRnHxlskq4Fkal9hAwqup9Lp43567iaPM4SBJJbksw8y10tucgxWYpRcJU41FAGSc6BJwty82nzbOQLA3i8M95Xyp7ppa5KK4t2ay6818v3zZtID6oX8jVvodDQVHF3QbbhIuKXi+mhTK1RlsCOZ/SBc7ABtPKYxBcsclZHW17qkGbhXZyaHjwtWtzScHNXBg4jLzT/W89vvS654/sp9OpmXkO0g1XGAPAk525JQPrre08gQvkiSi2D56JDsu8XvRkZd7utrzEbpx+3oBW5W07eGykcWd6vD6n15tJZD5o2AUcHr8SjmiVdxvg1iajNzDuoznMUMl/5pgl+m+IHbdNxO4lQ="
```

Comment:
What are these two different secure keys. Should we add a comment to what each of these are for?